src/EnergyPlus/Furnaces.cc

```diff
@@ -4739,11 +4737,7 @@ namespace Furnaces {
 
         if (!state.dataGlobal->DoingSizing && state.dataFurnaces->MySecondOneTimeFlag(FurnaceNum)) {
             // sizing all done.  check fan air flow rates
-            errFlag = false;
             thisFurnace.ActualFanVolFlowRate = state.dataFans->fans(thisFurnace.FanIndex)->maxAirFlowRate;
-            if (errFlag) {
-                ShowContinueError(state, format("...occurs in {} ={}", HVAC::unitarySysTypeNames[(int)thisFurnace.type], thisFurnace.Name));
-            }
```

Comment:
[src/EnergyPlus/Furnaces.cc:4744]:(style),[knownConditionTrueFalse],Condition 'errFlag' is always false